web/app/models/User.js

```diff
@@ -0,0 +1,16 @@
+// grab the mongoose module
+var mongoose = require("mongoose");
+
+// define our user model
+// module.exports allows us to pass this to other files when it is called
+var UserSchema = new mongoose.Schema({
+    userId: { type: Number, required: true },
+    email: { type: String, required: true },
+    firstName: { type: String, required: true },
+    lastName: { type: String, required: true },
+    userType: { type: String, required: true },
```

Comment:
Maybe we use a enum or class to define a set number of user types.  We would then use that type or class name here.  It is not a huge deal though.